spec/requests/admin/subscription_spec.rb

```diff
@@ -10,4 +10,46 @@
     end
   end
 
+  context "updating subscriptions" do
+    let(:order) {
+      order = create(:completed_order_with_totals, bill_address: address, ship_address: address)
+      order.line_items << create(:line_item, variant: product.master, interval: 2)
+     }
+    before do
+      product = FactoryGirl.create(:subscribable_product)
+      order = FactoryGirl.create(:completed_order_with_totals)
+      order.line_items << create(:line_item, variant: product.master, interval: 2)
+      CreateSubscriptionJob.new.perform(order.id)
+    end
+
+    scenario "can update a subscriptions address" do
+      visit spree.admin_path
+      click_link "Subscriptions"
+      click_link("Edit", visible: false)
+
+      fill_in("subscription_bill_address_attributes_address1", with: "updated address 1")
+      fill_in("subscription_bill_address_attributes_address2", with: "updated address 2")
+      fill_in("subscription_bill_address_attributes_city", with: "updated city")
+      fill_in("subscription_bill_address_attributes_zipcode", with: "11111")
+
+      fill_in("subscription_ship_address_attributes_address1", with: "updated address 1")
+      fill_in("subscription_ship_address_attributes_address2", with: "updated address 2")
+      fill_in("subscription_ship_address_attributes_city", with: "updated city")
+      fill_in("subscription_ship_address_attributes_zipcode", with: "11111")
+
+      click_button("Update")
+
+      expect(page).to have_field("subscription_bill_address_attributes_address1", with: "updated address 1")
+      expect(page).to have_field("subscription_bill_address_attributes_address2", with: "updated address 2")
+      expect(page).to have_field("subscription_bill_address_attributes_city", with: "updated city")
+      expect(page).to have_field("subscription_bill_address_attributes_zipcode", with: "11111")
```

Comment:
i didn't think that was necessary since we don't provide international shipping, and if the rest of the updates including state (which has a state name as well as state id) then theoretically the country should update fine as well